### PR TITLE
Update Attach Session API wrapper

### PR DESCRIPTION
**Pull request by Shalini-Subramanian**
### What does this Pull Request accomplish?

This Pull Request updates the `nisemidevicecontrol.py` example to attach to the existing SDC session using the newly created API `attach_to_existing_session` . 

### Why should this Pull Request be merged?
This Pull Request updates the wrapper to get the existing SDC session using the newly created API `attach_to_existing_session`.

### What testing has been done?
- Tested with `dut_communication_using_semi_device_control_nidigital`

### CODE REVIEW

File: nisdc/nisemidevicecontrol.py [hunk 1]
```diff
@@ -79,6 +79,17 @@ def __init__(self, isconfigpath):
         except Exception as e:
             print("Exception in accessing conf: {}".format(e))
             raise e
+        
+    def attach_to_existing_session(self):
+        """Returns the existing device control session
```
Comment (SowmyaRavee) [inline nisdc/nisemidevicecontrol.py:84]:
```suggestion
        """Returns the instantiated device control session if any
```

File: nisdc/nisemidevicecontrol.py [hunk 2]
```diff
@@ -79,6 +79,17 @@ def __init__(self, isconfigpath):
...
+        """
+        try:
+            self.semidevicecontrol_session = self.semidevicecontrol_main.AttachToExistingSession()
+            return self.semidevicecontrol_main
+        
+        except Exception as e:
+            print(e)
```
Comment (SowmyaRavee) [inline nisdc/nisemidevicecontrol.py:91]:
```suggestion
            print("Error occurred while attaching to the instantiated semi device control session: {}".foramat(e))
```

File: nisdc/nisemidevicecontrol.py [hunk 3]
```diff
@@ -79,6 +79,17 @@ def __init__(self, isconfigpath):
         except Exception as e:
             print("Exception in accessing conf: {}".format(e))
             raise e
+        
+    def attach_to_existing_session(self):
+        """Returns the existing device control session
+        """
+        try:
+            self.semidevicecontrol_session = self.semidevicecontrol_main.AttachToExistingSession()
+            return self.semidevicecontrol_main
```
Comment (SowmyaRavee) [inline nisdc/nisemidevicecontrol.py:88]: Is the required return object?

**Review by SowmyaRavee — APPROVED** [review]

**Review by SowmyaRavee — COMMENTED** [review]
Should we be returning the `self` object while attaching?

File: nisdc/nisemidevicecontrol.py (hunk 1 above)
Comment (Shalini-Subramanian) [reply nisdc/nisemidevicecontrol.py:84]: Updated

File: nisdc/nisemidevicecontrol.py (hunk 2 above)
Comment (Shalini-Subramanian) [reply nisdc/nisemidevicecontrol.py]: Updated

**Pull request merged**